azure-quantum/azure/quantum/optimization/online_problem.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from typing import TYPE_CHECKING, Union, Dict 
+from azure.quantum.optimization import Problem
+
+logger = logging.getLogger(__name__)
+
+__all__ = ['OnlineProblem']
+
+if TYPE_CHECKING:
+    from azure.quantum.workspace import Workspace
+
+class OnlineProblem(Problem):
+    def __init__(self, name:str, blob_uri:str):
+        super().__init__(self)
+        self.name = name
+        self.uploaded_blob_uri = blob_uri
+    
+    def evaluate(self, configuration: Union[Dict[int, int], Dict[str, int]]) -> float:
```

Comment:
Might need a similar error message for set_fixed_variables too I think.

Reply:
Also in line with the above comment, if the problem isn't downloaded, do we need to check & throw exceptions for all the existing functionality that rely on a downloaded problem?